integration/src/test/java/dev/ikm/tinkar/integration/langext/fhir/FhirTransformAPIIT.java

```diff
@@ -41,14 +52,40 @@ public class FhirTransformAPIIT extends TestHelper {
 
     @BeforeAll
     public void setup() {
-        loadSpinedArrayDataBase(SAP_DATASTORE_ROOT);
+        startSpinedArrayDataBase(new File(System.getProperty("user.home") + "/Solor/" + "snomedLidrLoinc-data-7-1-2024-withCollabData-dev"));
     }
 
+    @Test
+    public void testOwlSyntax() throws IOException {
```

Comment:
Both this test and the one below it don't appear to assert anything. Typically tests should have at least one assertion.